src/mixing.html

```diff
@@ -81,7 +82,9 @@ <h4 id="mixing_intent_grammar">The Grammar for <code class="attribue">intent</co
    </pre>
 
    <p>Here <a href="https://www.w3.org/TR/REC-xml-names/#NT-NCName"><code>NCName</code></a>
-   is as defined in  in [[xml-names]], and <code>digit</code> is a character in the range 0–9.</p>
+   is as defined in  in [[xml-names]],
+   <code>\pNd</code> is a character of class Nd (decimal digit),
```

Comment:
was `\pNd` meant to replace `\d` in the `number` rule? It hasn't yet.

Reply:
fixed